gameClientToGameWindowTests/errorMessage.spec.ts

```diff
@@ -47,6 +51,8 @@ readGames().forEach((game: string) => {
         });
 
         test('reality check', async () => {
+            // set a 3 minute timeout for this test
+            test.setTimeout(300000);
```

Comment:
minor suggestion: use underscores like the other timestamps in the code for consistency

Reply:
sure, done that